.github/workflows/examples.yml

```diff
@@ -101,3 +101,9 @@ jobs:
         done
       env:
         OMP_NUM_THREADS: 1
+    - name: Run PyTorch snapshot example
```

Comment:
[nit-picking] how about checkpoint instead of snapshot for the consistency?
```suggestion
    - name: Run PyTorch checkpoint example
```

Reply:
Thanks! I fixed.